src/commands/server/update.ts

```diff
@@ -38,6 +38,16 @@ export default class Update extends Command {
       description: 'Type of Kubernetes platform. Valid values are \"minikube\", \"minishift\", \"k8s (for kubernetes)\", \"openshift\", \"crc (for CodeReady Containers)\", \"microk8s\".',
       options: ['minikube', 'minishift', 'k8s', 'openshift', 'microk8s', 'docker-desktop', 'crc'],
     }),
+    domain: string({
```

Comment:
Can we make UX even smoother? 
I suppose now doman parameter is required and installation will be broker if an admin miss it during server:update for K8s, right?
If so - could we supply this option only when user needs to change it, otherwise - the initially configure value could be untouched?

Reply:
> If so - could we supply this option only when user needs to change it

I don't understand